Integrations/MicrosoftGraphUser/MicrosoftGraphUser.yml

```diff
@@ -217,9 +217,11 @@ script:
       name: properties
       required: false
       secret: false
-    deprecated: false
-    description: Gets newly created, updated, or deleted users without performing
-      a full read of the entire user collection.
+    deprecated: true
+    description: DEPRECATED  - We recommend not useing this command since it will
+      return a single page only, it is equivalent to msgraph-user-list command - Gets
+      newly created, updated, or deleted users without performing a full read of the
+      entire user collection.
```

Comment:
Deprecated. This command only returns a single page. Use the msgraph-user-list command instead, which gets newly created, updated, or deleted users without performing a full read of the entire user collection.